src/createTokenBridgeEnoughCustomFeeTokenAllowance.ts

```diff
@@ -0,0 +1,33 @@
+import { Address, PublicClient } from 'viem';
+
+import { tokenBridgeCreator } from './contracts';
+import { validParentChainId } from './types/ParentChain';
+import { fetchAllowance } from './utils/erc20';
+import { createTokenBridgeEstimatedFees } from './constants';
+
+export type CreateTokenBridgeEnoughCustomFeeTokenAllowanceParams = {
+  nativeToken: Address;
+  account: Address;
+  publicClient: PublicClient;
+};
+
+export async function createTokenBridgeEnoughCustomFeeTokenAllowance({
+  nativeToken,
+  account,
```

Comment:
Should we be a bit more verbose in these variables? For example, this is the token bridge deployer

Reply:
Maybe change to `owner`